openapi.json

```diff
@@ -709,6 +710,13 @@
                                         "format": "float",
                                         "minimum": 0
                                     },
+                                    "route": {
+                                        "type": "array",
+                                        "items": {
+                                            "type": "string",
+                                            "pattern": "^((0?[1-9]|[1-2]\\d|3[0-6])[LCR]?)|([A-Z]{5})|([A-Z]{3})|([A-Z]{1-3})$"
```

Comment:
Not all routes will conform to this regex. Does the application also have this regex or is this just API related (and VAs can change this?) @GenericNerd

Reply:
This is only for the API specification to be able to generate routing that is loosely correct. The actual API code only expects an array of strings in this case.